### PR TITLE
Fix/Remove-form-styles

### PR DESCRIPTION
**Pull request by brittrich**
@bmartinez287 This removes the form template changes that had a more global impact. I'll add a few different fixes for the dept directory view form as I work on form styles for the other profile directory as well.

### CODE REVIEW

File: apps/drupal-default/particle_theme/templates/form/views-exposed-form.html.twig [hunk 1]
```diff
@@ -16,6 +16,6 @@
   #}
 {{ q }}
 {% endif %}
-<div class="form-inline clearfix mb-3 xl:w-1/3">
+<div class="form-inline clearfix">
```
Comment (bmartinez287) [inline apps/drupal-default/particle_theme/templates/form/views-exposed-form.html.twig]: Screen Shot 2021-03-08 at 2 24 15 PM
flow-root > clearfix
https://tailwindcss.com/docs/upgrading-to-v2
Idk if the change makes sense in this scenario but as a heads up.

**Comment by brittrich** [comment]
Added flow-root, made sense to go ahead and update since it does the exact same thing and is what we are moving toward.

**Comment by brittrich** [comment]
@bmartinez287 can I merge this in myself, or does someone else need to?

**Comment by bmartinez287** [comment]
@brittrich you can merge it yourself. That's one benefit of having at a separate repo it does not affect utccloud until the particle hash is updated to include this change or other ones related to it. 
The only request is that you try to take 5 minutes to test that it does not interfere/collide with other systems.

**Comment by bmartinez287** [comment]
Since, I'm already looking at this issue. I will merge it in but feel free to merge in a  Particle PR if is ready and you feel confident on it.

**Pull request merged**